x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/span_links/span_links.spec.ts

```diff
@@ -7,505 +7,545 @@
 import expect from '@kbn/expect';
 import { ProcessorEvent } from '@kbn/observability-plugin/common';
 import { Readable } from 'stream';
-import type { ApmSynthtraceEsClient } from '@kbn/apm-synthtrace';
+import type { ApmSynthtraceEsClient, ApmSynthtracePipelines } from '@kbn/apm-synthtrace';
+import moment from 'moment';
 import type { DeploymentAgnosticFtrProviderContext } from '../../../../ftr_provider_context';
 import { generateSpanLinksData } from './data_generator';
 
 export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderContext) {
   const apmApiClient = getService('apmApi');
   const synthtrace = getService('synthtrace');
 
-  const start = new Date('2022-01-01T00:00:00.000Z').getTime();
-  const end = new Date('2022-01-01T00:15:00.000Z').getTime() - 1;
+  const baseTime = Date.now() - 1;
+  const start = moment(baseTime).subtract(15, 'minutes');
+  const end = moment(baseTime);
 
-  describe('contains linked children', () => {
-    let ids: ReturnType<typeof generateSpanLinksData>['ids'];
-    let apmSynthtraceEsClient: ApmSynthtraceEsClient;
+  const scenarios: ApmSynthtracePipelines[] = ['default', 'apmToOtel'];
 
-    before(async () => {
-      const spanLinksData = generateSpanLinksData();
-      apmSynthtraceEsClient = await synthtrace.createApmSynthtraceEsClient();
+  // skips all tests that validate the `transaction.id` in `span` docs for Otel
```

Comment:
Relates to: [#214557](https://github.com/elastic/kibana/issues/214557)

Reply:
This should be fixed in the [issue](https://github.com/elastic/kibana/issues/214557)'s PR, right? Or should we open a new issue to update the tests to not rely on `transaction.id`?

Reply:
It depends on the outcome of the issue you mentioned. If  `transaction.id` won't be supported, then that PR will have to adjust to not rely on `transaction.id`, otherwise, we unskip these tests.